servers/auth-server/src/main/webapp/WEB-INF/ups-realm.json

```diff
@@ -61,9 +61,9 @@
             "name": "unified-push-server-js",
             "enabled": true,
             "publicClient": true,
-            "baseUrl": "/ag-push",
+            "baseUrl": "http://localhost:8082/ag-push",

```

Comment:
@sebastienblanc any comment on my comment above> that the `auth` will go away, since it's completely decoupled